src/routes/user.route.ts

```diff
@@ -4,10 +4,10 @@ import {
   userSignup,
   userLogin,
   changeAccountStatus,
-  updatePassword
+  userLogout,
 } from "../controllers/user.controller";
-import { protectRoute, restrictTo } from "../middlewares/auth.middleware";
-import { validateUser, validateUserLogin,validateUserUpdatePassword } from "../validations/user.validate";
```

Comment:
Why did you delete this route?


I think you can put it back because is important

Reply:
> Why did you delete this route?
> 
> I think you can put it back because is important

Resolved sir